tests/xbm/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+IF(XBM_FOUND)
```

Comment:
there is no `XBM_FOUND` symbol in the cmake code.  looks like we always build the xbm code, so you'll want to drop this if statement.